### DISCUSSION TITLE
Off-topic: Please teach me your ways senpai

### DISCUSSION THREAD

**Discussion by thelinuxlich** [graphql-editor/graphql-zeus#353] [General]
I understand you solved performance issues and avoided the hardcoded instantiation depth limit in Typescript. I wish to apply your wisdom to the projects Kysely and Zodios, as both are facing similar issues.

**Comment by aexol** [comment]
Hi @thelinuxlich 
Feel free to message me 
https://twitter.com/ACzemiel

The first thing is to remember TS has a compiler. Sometimes you need to use generics just for compiler to know it need to wait for an user input

**Reply by aexol** [reply]
https://discord.gg/bHf2cw8e
Also you can join our discord
